db/migrate/20241111114155_add_status_to_payroll_runs.rb

```diff
@@ -0,0 +1,7 @@
+class AddStatusToPayrollRuns < ActiveRecord::Migration[7.0]
+  def change
+    add_column :payroll_runs, :status, :string, default: "pending", null: false
+
+    PayrollRun.update_all(status: "complete")
```

Comment:
There won't be any inflight payroll runs when this rolls out so it's safe to update all the existing ones to completed